README.md

```diff
@@ -36,11 +35,28 @@ choco install ffmpeg
 # on Windows using Scoop (https://scoop.sh/)
 scoop install ffmpeg
 ```
-
+2. Diart requires some packages to be installed via [`Conda`](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html)
```

Comment:
You have double the numbering (two step #2). I hope this step will be removed though, as we won't need Conda